### PR TITLE
Final announcement

### PR DESCRIPTION
**Pull request by regenscheid**
These will need to be updated with final dates and links once the document is approved.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
 
 ## Available Online
 
-[Draft FIPS 201-3 Personal Identity Verification (PIV) of Federal Employees and Contractors](_FIPS201/abstract.md)
+[FIPS 201-3 Personal Identity Verification (PIV) of Federal Employees and Contractors](_FIPS201/abstract.md)
 
-## Comments
-The draft of FIPS 201-3 is available for review and comment on the NIST Pages website at <https://pages.nist.gov/FIPS201/>.
+## Summary of Changes
+FIPS 201-2 addresses the comments received during the public comment period in November 2020.  High level changes include:
```
Comment (jricher) [inline index.md:17]:
```suggestion
FIPS 201-3 addresses the comments received during the public comment period in November 2020.  High level changes include:
```

**Comment by jricher** [comment]
README will need a link to pages.nist.gov someplace, also.

File: index.md [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
-the deadline will be published electronically at <https://github.com/usnistgov/FIPS201> without change or redaction, so commenters 
-should not include information they do not wish to be posted. Personal information, such as account numbers or Social Security 
-numbers, or names of other individuals, should not be included. Do not submit confidential business information or otherwise sensitive 
-or protected information. Comments that contain profanity, vulgarity, threats, or other inappropriate language or content will not 
-be posted or considered.
+A detailed list of changes is available in [FIPS 201-3, Appendix E, Revision History](_FIPS201/revisions.md), and all comments and dispositions can be found on the [project repository](https://github.com/usnistgov/FIPS201/issues) or in the [2020 Draft comments and dispositions](TBD).
 
 ## More Information:
-A printable PDF is available on the NIST Computer Security Resource Center at <https://csrc.nist.gov/publications/detail/fips/201/3/draft>.
+A printable PDF is available on the NIST Computer Security Resource Center at <https://csrc.nist.gov/publications/detail/fips/201/3/>.
```
Comment (regenscheid) [inline index.md:28]:
```suggestion
A printable PDF is available on the NIST Computer Security Resource Center at <https://csrc.nist.gov/publications/detail/fips/201/3/final>.
```

File: index.md [hunk 3]
```diff
@@ -1,34 +1,30 @@
...
-
-All submissions, including attachments and other supporting materials, will become part of the public record and subject to public 
-disclosure. NIST reserves the right to publish relevant comments, unedited and in their entirety. Relevant comments received by 
-the deadline will be published electronically at <https://github.com/usnistgov/FIPS201> without change or redaction, so commenters 
-should not include information they do not wish to be posted. Personal information, such as account numbers or Social Security 
-numbers, or names of other individuals, should not be included. Do not submit confidential business information or otherwise sensitive 
-or protected information. Comments that contain profanity, vulgarity, threats, or other inappropriate language or content will not 
-be posted or considered.
+A detailed list of changes is available in [FIPS 201-3, Appendix E, Revision History](_FIPS201/revisions.md), and all comments and dispositions can be found on the [project repository](https://github.com/usnistgov/FIPS201/issues) or in the [2020 Draft comments and dispositions](TBD).
```
Comment (regenscheid) [inline index.md:25]:
```suggestion
A detailed list of changes is available in [FIPS 201-3, Appendix E, Revision History](_FIPS201/revisions.md), and all comments and dispositions can be found on the [project repository](https://github.com/usnistgov/FIPS201/issues) or in the [2020 Draft comments and dispositions](https://csrc.nist.gov/publications/detail/fips/201/3/final).
```

Placeholder for now until we can do a direct link to the comments PDF

File: index.md [hunk 4]
```diff
@@ -1,34 +1,30 @@
...
 layout: cover
-title: "Draft FIPS 201-3 Available for Public Comment"
-description: "Draft FIPS 201-3 Personal Identity Verification (PIV) of Federal Employees and Contractors"
+title: "FIPS 201-3 Personal Identity Verification (PIV) of Federal Employees and Contractors"
+description: "FIPS 201-3 Personal Identity Verification (PIV) of Federal Employees and Contractors"
 permalink: /
 ---
 
-## Posted: November, 2020
+## Posted: XXX, XX 2021
```
Comment (jricher) [inline index.md:8]:
```suggestion
## Posted: Jan 24, 2022
```

File: README.md [hunk 5]
```diff
@@ -1,21 +1,16 @@
...
+# FIPS 201-3: Personal Identity Verification (PIV) of Federal Employees and Contractors
+The National Institute of Standards and Technology (NIST) is pleased to announce the approval of Federal Information Processing Standard (FIPS) Publication 201-3, Personal Identity Verification of Federal Employees and Contractors. (See the Federal Register Notice announcing FIPS 201-3 approval.)
```
Comment (jricher) [inline README.md:2]:
```suggestion
The National Institute of Standards and Technology (NIST) is pleased to announce the approval of Federal Information Processing Standard (FIPS) Publication 201-3, Personal Identity Verification of Federal Employees and Contractors. (See the Federal Register Notice announcing FIPS 201-3 approval.)

The rendered version is available from NIST pages <https://pages.nist.gov/FIPS201/>.
```

File: README.md [hunk 6]
```diff
@@ -1,21 +1,18 @@
...
```
Comment (regenscheid) [inline README.md:2]:
```suggestion
The National Institute of Standards and Technology (NIST) is pleased to announce the approval of Federal Information Processing Standard (FIPS) Publication 201-3, Personal Identity Verification of Federal Employees and Contractors. See the Federal Register Notice announcing FIPS 201-3 approval at <https://www.federalregister.gov/d/2022-01246>.
```

File: index.md [hunk 7]
```diff
@@ -1,34 +1,30 @@
...
+title: "FIPS 201-3 Personal Identity Verification (PIV) of Federal Employees and Contractors"
+description: "FIPS 201-3 Personal Identity Verification (PIV) of Federal Employees and Contractors"
 permalink: /
 ---
 
-## Posted: November, 2020
+## Posted: Jan 24, 2022
 
-The National Institute of Standards and Technology (NIST) requests comments on draft Federal Information Processing Standard (FIPS) 201-3, Personal Identity Verification (PIV) of Federal Employees and Contractors. This revision proposes several revisions to support the policy objectives outlined in [OMB M-19-17](https://www.whitehouse.gov/wp-content/uploads/2019/05/M-19-17.pdf), align with emerging standards and technologies for digital identity, support the use of alternative authenticators, and encourage interoperability through federation.
+The National Institute of Standards and Technology (NIST) is pleased to announce the approval of Federal Information Processing Standard (FIPS) Publication 201-3, Personal Identity Verification of Federal Employees and Contractors. (See the Federal Register Notice announcing FIPS 201-3 approval.)
```
Comment (regenscheid) [inline index.md:10]:
```suggestion
The National Institute of Standards and Technology (NIST) is pleased to announce the approval of Federal Information Processing Standard (FIPS) Publication 201-3, Personal Identity Verification of Federal Employees and Contractors. See the Federal Register Notice announcing FIPS 201-3 approval at <https://www.federalregister.gov/d/2022-01246>.
```

**Pull request merged**